examples/2d/elixir_ape_gauss.jl

```diff
@@ -0,0 +1,60 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the acoustic perturbation equations
+
+v_mean = (0.25, 0.25)
+rho_mean = 1.0
+c_mean = 1.0
+equations = AcousticPerturbationEquations2D(v_mean, rho_mean, c_mean)
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+solver = DGSEM(3, flux_lax_friedrichs)
+
+coordinates_min = (-1, -1) # minimum coordinates (min(x), min(y))
+coordinates_max = ( 1,  1) # maximum coordinates (max(x), max(y))
+
+# Create a uniformely refined mesh with periodic boundaries
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level=4,
+                n_cells_max=30_000) # set maximum capacity of tree data structure
+
+# A semidiscretization collects data structures and functions for the spatial discretization
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition_gauss, solver)
+
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+# Create ODE problem with time span from 0.0 to 1.0
+ode = semidiscretize(semi, (0.0, 1.0));
```

Comment:
```suggestion
ode = semidiscretize(semi, (0.0, 1.0))
```

Reply:
Done in 284d1ecce8a5ed2158fb767b1613812bf571f5db.